### ISSUE TITLE
Manual installation does not work in Magento 2.3.1

### ISSUE THREAD

**Issue by kanenas** [simplifycom/simplify-magento-module#9]
**Magento**: 2.3.1
**php**: 7.2.28

I have followed the instructions from `README.md`, but the extension is not showing in `Stores > Configuration > Sales > Payment Methods`.
No errors!

I have also done:
- `./bin/magento setup:upgrade`
- `./bin/magento cache:clean`
- `./bin/magento setup:di:compile`
- `./bin/magento setup:static-content:deploy en_US el_GR -f`
- `./bin/magento cache:clean`

Still nothing.

**Comment by kevinsh25** [comment]
Faced same issue. Installed extension under app/code/ and was able to fetch payment method under admin.
However on enabling it, unable to get any payment methods on frontend

**Comment by ioweb-gr** [comment]
Same issue when installing via composer for magento 2.2.11

**Comment by rudmarv** [comment]
Hi @ioweb-gr, I just wanna ask if you have manage to solve the problem in the front-end? Thanks!

**Comment by ioweb-gr** [comment]
Nope unfortunately. You wouldn't expect such issues being unreplied for so long. Our customer is still using another bank's gateway instead of this one because of this issue.

**Comment by JohnbConneely** [comment]
Hey @ioweb-gr, I have raised this issue with the developers. Could you also raise this through the banks support team so that it can be tracked correctly? This issue tab is not being actively monitored and will be closed soon so that we can properly monitor support tickets through the support queue.

**Comment by ioweb-gr** [comment]
I actually have done so months ago for both issues I've mentioned here. The current one and the frame options header. I regret to inform you that we've been totally ignored and a few months later  we only received a canned response with the links to the repositories for magento and the docs to implement ourselves.

**Comment by JohnbConneely** [comment]
@ioweb-gr , can you give me the support ticket number that was raised with this? Should be in the email header of the response they sent to you.

**Comment by ioweb-gr** [comment]
The issue was raised via phone support so there's no ticket number unfortunately. The last communication we received was when our eshop owner contacted the sales department of NBG and then they replied to them again with the message I told you about earlier. 

A repeat to use the plugins found in github or to code it via the API docs. 

As you can understand we've pretty much hit a dead end with NBG regarding this which only seems to be the continuation of their previous ideas of technical support.

**Comment by kanenas** [comment]
This is the answer (sorry it's in Greek) from the **IT e-Commerce Support Team** (**Tue 23/06/2020 12:48**):

> Καλησπέρα σας, 
> 
> Αναφορικά με το Simplify και το magento 2 που υποστηρίζει θα θέλαμε να σας ενημερώσουμε ότι το 2.1 δεν υποστηρίζεται και το 2.3 δεν υποστηρίζεται (τουλάχιστον οχι ακόμη). Το version 2.2 υποστηρίζεται κανονικά. Απολογούμαστε για όλη την καθυστέρηση.
> Η ΕΤΕ σας προτείνει να χρησιμοποιήσετε τη μέθοδο hosted payments (μόνο χωρίς SDK ώστε να συμβαδίζει με την σύμβασή σας), πληροφορίες θα βρείτε παρακάτω:
> 
> https://github.com/myNBGcode/E-commerceSimplify#Hosted-checkout.
> https://ibanknbg.simplify.com/commerce/docs/tools/hosted-payments.
> 
> Στη διάθεσή σας,
> 
> IT e-Commerce Support Team | Εθνική Τράπεζα της Ελλάδος |
> e-mail: itecommercesupport@nbg.gr    | web: www.nbg.gr

go figure!

**Comment by ioweb-gr** [comment]
The message says basically that Magento 2.1 / 2.3 is not supported and it only works in 2.2 which is EOL now and that NBG suggest to use hosted payments without sdk. 

The thing is that we tried on 2.2 and we faced the same issues as in 2.3.1. And currently 2.2 is EOL so this module should be made compatible with at least 2.3.5-p1 if not 2.4.

2.4 contains a CSP which will definitely break this module.
